docs/src/modules/components/overview/charts/ChartsCommunityOrPro.tsx

```diff
@@ -19,16 +19,16 @@ export default function ChartsCommunityOrPro() {
           caption={'Community and Pro'}
           title={'Two packages for every need'}
           description={
-            'Start with the free-forever Community version, then upgrade to Pro when you are ready for additional features and components.'
+            "Start with the free-forever Community version, then upgrade to Pro when you're ready for additional features and components."
           }
           communityDescription={
-            'Free forever under MIT license. Includes essential charts and core features like axes, legends, styling customizations, tooltips, highlights, and more.'
+            'Free forever under an MIT license. Includes essential charts and core features like axes, legends, styling customizations, tooltips, highlights, and more.'
           }
           proDescription={
-            'Requires a commercial license. Includes advanced charts and features like zoom and pan, chart export. Better suited for data rich and enterprise applications.'
+            'Requires a commercial license. Includes advanced charts and features such as zooming, panning, and exporting. Ideal for data-rich and enterprise applications.'
           }
           // premiumDescription={
-          //   'Requires a commercial license. better suited for data rich and enterprise applications, including features such as reordering, lazy loading or virtualization.'
+          //   'Requires a commercial license. Includes the most advanced features such as reordering, lazy loading, and virtualization.'
```

Comment:
Just to be sure when premium get released we don't include data grid features
```suggestion
          //   'Requires a commercial license. Includes the most advanced features such as ... TO COMPLET.'
```